src/main/java/com/vackosar/gitflowincrementalbuild/boundary/UnchangedProjectsRemover.java

```diff
@@ -37,53 +37,51 @@
     private Logger logger = LoggerFactory.getLogger(UnchangedProjectsRemover.class);
 
     @Inject private ChangedProjects changedProjects;
-    @Inject private MavenSession mavenSession;
-    @Inject private Configuration.Provider configProvider;
 
-    void act() throws GitAPIException, IOException {
+    void act(Configuration config) throws GitAPIException, IOException {
```

Comment:
Codacy found an issue: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/gitflow-incremental-builder/gitflow-incremental-builder/pullRequest?prid=5908979)